src/liboslexec/shadingsys.cpp

```diff
@@ -1888,6 +1927,14 @@ ShadingSystemImpl::getstats (int level) const
 
 
 
+uint64_t
+ShadingSystemImpl::lookup_string_tag (ustring str)
+{
+    return m_tag_map.count(str) ? m_tag_map[str] : StringTags::UNKNOWN_STRING;
+}
```

Comment:
```
auto itr = m_tag_map.find(str);
return itr != m_tag_map.end() ? itr->second : StringTags::UNKNOWN_STRING;

```

Reply:
Cool, I'll integrate your suggestions if the alternative representation does not pan out.